### PR TITLE
SlapdObject command methods return process data

### PR DESCRIPTION
**Pull request by azmeuk**
A simple refactoring, so the output of commands is usable.

### CODE REVIEW

**Review by droideck — DISMISSED** [review]
A very small nitpick... 
Could you please use an imperative mood in the subject line? It makes the changelog much more readable.

Besides that, you have my ack!

**Comment by azmeuk** [comment]
Something like "Make SlapdObject commands return their outputs" would fit better?

**Comment by encukou** [comment]
Before the API is set in stone, are we sure it's the right one? Would it be better to return something like [`subprocess.CompletedProcess`](https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess)?

Could you also add the return value to the docstrings and documentation?

**Comment by azmeuk** [comment]
@encukou I did not know about `CompletedProcess` but this seems the cleaner way to do this!
I have fixed the patch (sorry for the force-push).

File: Lib/slapdtest/_slapdtest.py [hunk 1]
```diff
@@ -512,60 +512,69 @@ def _cli_popen(self, ldapcommand, extra_args=None, ldap_uri=None,
...
+        if proc.stdout is not None:
+            self._log.debug('stdout=%s', proc.stdout.decode("utf-8"))
+        if proc.stderr is not None:
+            self._log.debug('stderr=%s', proc.stderr.decode("utf-8"))
+        if proc.returncode != 0:
             raise RuntimeError(
-                '{!r} process failed:\n{!r}\n{!r}'.format(
-                    args, stdout_data, stderr_data
-                )
+                'Process failed: {!r}'.format(" ".join(args))
```
Comment (azmeuk) [inline Lib/slapdtest/_slapdtest.py:528]: I removed `stdout` and `stderr` here as they have been logged a few lines ago, and large outputs as exceptions messages are totally unpracticable.

**Review by droideck — APPROVED** [review]
Looks clean and improved!
You have my ack.

**Comment by azmeuk** [comment]
I included this patch on `python-slapd`.
https://github.com/python-ldap/python-ldap/issues/203#issuecomment-731778829

**Pull request closed without merging**